From 38f9fdfc49f9cb6e6d62e10cfb00b3326ab0f2c5 Mon Sep 17 00:00:00 2001
From: Kamil Debski <k.debski@samsung.com>
Date: Tue, 21 May 2013 17:32:17 +0200
Subject: [PATCH 0315/1302] modem_if: Remove linux/cma.h use

Remove linux/cma.h from modem_modemctl_device_xmm6262.c
It wasn't used anyway and was included for linux/rbtree.h only.

Signed-off-by: Kamil Debski <k.debski@samsung.com>
Signed-off-by: Kyungmin Park <kyungmin.park@samsung.com>
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
---
 drivers/misc/modem_if/modem_modemctl_device_xmm6262.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/misc/modem_if/modem_modemctl_device_xmm6262.c b/drivers/misc/modem_if/modem_modemctl_device_xmm6262.c
index e400046..311836c 100644
--- a/drivers/misc/modem_if/modem_modemctl_device_xmm6262.c
+++ b/drivers/misc/modem_if/modem_modemctl_device_xmm6262.c
@@ -22,7 +22,6 @@
 #include <linux/gpio.h>
 #include <linux/delay.h>
 #include <linux/platform_device.h>
-#include <linux/cma.h>
 #include <plat/devs.h>
 #include <linux/platform_data/modem.h>
 #include "modem_prj.h"
-- 
1.8.3.2

